.travis.yml

```diff
@@ -16,7 +14,7 @@ env:
 before_install:
   - go get golang.org/x/tools/cmd/cover
 script:
-  - go test ./... -race -coverprofile=coverage.txt -covermode=atomic
+  - go test . -race -coverprofile=coverage.txt -covermode=atomic
```

Comment:
Just a thought, but is it worth expanding the tests to include an explicit test of the behaviour you're expecting a user to see, with respect to import paths? That suite of tests might also include your expectations on GOPATH vs module mode in Go 1.11.